src/coreclr/src/jit/lower.cpp

```diff
@@ -5777,22 +5776,22 @@ void Lowering::ContainCheckLclHeap(GenTreeOp* node)
 // Arguments:
 //    node - pointer to the node
 //
-void Lowering::ContainCheckRet(GenTreeOp* ret)
+void Lowering::ContainCheckRet(GenTreeUnOp* ret)
 {
     assert(ret->OperIs(GT_RETURN));
 
 #if !defined(TARGET_64BIT)
     if (ret->TypeGet() == TYP_LONG)
     {
-        GenTree* op1 = ret->gtGetOp1();
+        GenTree* op1 = ret->gtOp1;
```

Comment:
Seems odd to go back to direct data member access. It was, is and will always be a bad programming practice for common data structures such as IR ones.

Reply:
You are right, I forgot that `gtGetOp1` is declared for `GenTree`, not for `GenTreeOp`). Will revert that.
Thank you.